internal/console/helpers/helpers.go

```diff
@@ -30,6 +32,24 @@ type ProgressBar struct {
 	progress chan float64
 }
 
+// Printer wil print console output with colors
+// Medium is for medium sevevity results
+// High is for high sevevity results
+// Low is for low sevevity results
+// Info is for info sevevity results
+// Success is for successful prints
+// Line is the color to print the line with the vulnerability
+// minVersion is a bool that if true will print the results output in a minimum version
```

Comment:
would change this reference to something closer to the flag "minimal", to be coherent

Reply:
resolved